docs/user/running-cincinnati.md

```diff
@@ -16,6 +16,34 @@ The Cincinnati deploymentconfig is available [here](../../dist/openshift/cincinn
 oc new-project cincinnati
 ```
 
+#### Create secret for container registry
+
+If Cincinnati needs credentials to access a container registry to fetch the release payload information , you need to create a secret with name `cincinnati-credentials`. Make sure the auth for the container registry is present in the config.json file.
+
+```shell
+oc create secret generic cincinnati-credentials --from-file=.dockerconfigjson=/home/lmohanty/.docker/config.json --type=kubernetes.io/dockerconfigjson
+```
+
+An example config.json
+
+```
+$ cat ~/.docker/config.json
+{
+        "auths": {
+                "https://index.docker.io/v1/": {
+                        "auth": "foo="
+                },
+                "quay.io": {
+                        "auth": "bar=="
+                }
+        },
+        "HttpHeaders": {
+                "User-Agent": "Docker-Client/19.03.12 (linux)"
+        }
+```
+
+For more information around use of the container registry refer section on [configuring a container registry](#configure-a-container-registry-to-scrape-release-payload-information).
```

Comment:
nit: rewording:

> For more information around use of the container registry, see [the section on configuring a container registry](#configure-a-container-registry-to-scrape-release-payload-information).

Reply:
Fixed in https://github.com/openshift/cincinnati/pull/296/commits/57b9b01ed74d0a80299ea81a2d01047f1b6b6bc8